NST/src/main/java/com/ebay/nst/NSTServiceTestRunner.java

```diff
@@ -62,9 +60,9 @@ public default void run(IHookCallBack hookCallback, ITestResult iTestResult) {
 			iTestResult.setThrowable(t);
 			iTestResult.setStatus(ITestResult.FAILURE);
 		}
-		
+
 		// If the test passed write mocks/tests
-		if (iTestResult.getStatus() == ITestResult.SUCCESS) {
+		if (iTestResult.getStatus() == ITestResult.SUCCESS || iTestResult.getStatus() == ITestResult.STARTED) {
```

Comment:
Can you please add a comment explaining why we are checking for OR ITestResult.STARTED ?